ceph/ceph.py

```diff
@@ -2096,6 +2096,17 @@ def upload_file(self, src, dst, sudo=False):
         client = self.rssh if sudo else self.ssh
         client().open_sftp().put(src, dst)
 
+    def download_file(self, src, dst, sudo=False):
```

Comment:
Call same method for CephObject as well. Refer - https://github.com/red-hat-storage/cephci/blob/master/ceph/ceph.py#L2153

Reply:
Added